src/index.html

```diff
@@ -5,10 +5,63 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#" class="logo__link">
+        <img src="images/logo.png"
+             alt="MOYO Logo" class="image">
+      </a>
+      <nav class="nav">
+        <ul class="list">
```

Comment:
Use BEM convention naming if this is the element of nav so this should be nav__list

Reply:
BEM is a theme of next lesson, not previous. But Ok